awscli/customizations/eks/get_token.py

```diff
@@ -90,80 +110,93 @@ def _run_main(self, parsed_args, parsed_globals):
 
     def discover_api_version(self):
         """
-        Parses the KUBERNETES_EXEC_INFO environment variable and returns the API
-        version. If the environment variable is empty, malformed, or invalid,
-        return the v1alpha1 response and print an message to stderr.
+        Parses the KUBERNETES_EXEC_INFO environment variable and returns the
+        API version. If the environment variable is empty, malformed, or
+        invalid, return the v1alpha1 response and print a message to stderr.
 
         If the v1alpha1 API is specified explicitly, a message is printed to
         stderr with instructions to update.
 
         :return: The client authentication API version
         :rtype: string
         """
-        alpha_api = "client.authentication.k8s.io/v1alpha1"
-        beta_api = "client.authentication.k8s.io/v1beta1"
-        v1_api = "client.authentication.k8s.io/v1"
-        # At the time Kubernetes v1.29 is released upstream (aprox Dec 2023),
+        # At the time Kubernetes v1.29 is released upstream (approx Dec 2023),
         # "v1beta1" will be removed. At or around that time, EKS will likely
         # support v1.22 through v1.28, in which client API version "v1beta1"
         # will be supported by all EKS versions.
-        fallback_api_version = alpha_api
+        fallback_api_version = ALPHA_API
 
         error_prefixes = {
             "error": "Error parsing",
             "empty": "Empty",
         }
 
-        error_msg_tpl = ("{0} KUBERNETES_EXEC_INFO, defaulting "
-                    "to {1}. This is likely a bug in your Kubernetes "
-                    "client. Please update your Kubernetes client.")
-        unrecognized_msg = ("Unrecognized API version in KUBERNETES_EXEC_INFO, defaulting "
-                    "to {0}. This is likely due to an outdated AWS CLI."
-                    " Please update your AWS CLI.".format(fallback_api_version))
-        deprecation_msg_tpl = ("Kubeconfig user entry is using deprecated API "
-                    "version {0}. Run 'aws eks update-kubeconfig' to update")
+        error_msg_tpl = (
```

Comment:
It would be great if we can move the templates up as module level constants as well to make it easier to find the wording and reduce the size of the core logic for this function.